controllers/storagecluster/cephtoolbox.go

```diff
@@ -99,3 +116,22 @@ func removeDuplicateTolerations(tolerations []corev1.Toleration) []corev1.Tolera
 	}
 	return list
 }
+
+func getMultusPublicNetwork(sc *ocsv1.StorageCluster) (string, error) {
+	err := validateMultusSelectors(sc.Spec.Network.Selectors)
+	if err != nil {
+		return "", err
```

Comment:
It seems like maybe this is a reflection of ocs-operator's codebase, but I'm curious why most errors are just returned without any additional info (e.g., using %w to wrap errors). Does the ocs-operator team find that this makes debugging failures more difficult?

Reply:
Hello Blaine, I wanted to mention that we aren't currently mandating contributors to encapsulate a message or include additional logs. I believe this is something we should consider implementing in the future as it could enhance the ease of debugging.